src/state/slices/swappersSlice/swappersSlice.ts

```diff
@@ -102,7 +106,45 @@ export const swappersApi = createApi({
         return { data: maybeQuote }
       },
     }),
+    getThorTradeQuote: build.query<
```

Comment:
Instead of having all swapper endpoints in one file, I suspect we'll want to keep them logically separated into separate files to better maintainability.

See `src/state/apis/swapper/swapperApi.ts` for an example implementation of this pattern where we use the `injectEndpoints` method to add additional endpoints (`getAvailableSwappers` and `getIsTradingActive`).

Reply:
🧠